### ISSUE TITLE
wotmad.herokuapp.com seems to be down

### ISSUE THREAD

**Issue by huntwj** [avidal/wotmad#1]
I don't know if you're the one maintaining that site, but if you are...

- Visit http://wotmad.herokuapp.com/scripts/
- Enjoy Application Error

I'm happy to help with the maintenance if time is an issue. Let me know how I can help.

**Comment by avidal** [comment]
Yeah, it's actually been down for awhile. Our stat database eclipsed the free tier in Heroku so we had to switch off...and then the database was lost by the provider. I don't play anymore and I've lost interest in supporting any tooling. If you want the repository I can transfer it to you as well as the app on heroku.
